src/components/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,238 @@
+      if (isRemainActive !== undefined && isRemainActive !== null) {
+        onPressButton && onPressButton(isRemainActive)
+      } else {
+        setIsActive(!isActive)
+        onPressButton && onPressButton(isActive)
+      }
+    }
+
+    const renderLabelText = () =>
+      textComponent ||
+      (text ? (
+        <LabelTextView disable={!!disable} disableOpacity={disableOpacity} style={textContainerStyle}>
+          <LabelText style={textStyle}>{text}</LabelText>
+        </LabelTextView>
+      ) : null)
+
+    const handleLayout = (event: LayoutChangeEvent) => {
+      const {width, height} = event.nativeEvent.layout
+      widthBounceableRef.value = width
+      heightBounceableRef.value = height
+    }
+
+    return (
+      <RadioButtonWrapper testID="container" style={wrapperStyle}>
+        <BounceableStyle
+          outer={outer}
+          disable={!!disable}
+          disableOpacity={disableOpacity}
+          ringColor={ringColor}
+          testID="bounceable"
+          ref={ref}
+          disabled={disable}
+          onLayout={handleLayout}
+          style={style}
+          onPress={handlePress}
+          {...props}>
+          <RadioButtonInnerContainer
+            maxWidth={widthBounceableRef.value}
+            maxHeight={heightBounceableRef.value}
+            inner={inner}
+            isActive={isActive}
+            innerBackgroundColor={innerBackgroundColor}
+            style={innerContainerStyle}
+            testID="circle"
+          />
+        </BounceableStyle>
+        {renderLabelText()}
+      </RadioButtonWrapper>
+    )
+  },
+)
+
+RadioButton.displayName = 'RadioButton'
+
+export default RadioButton
+
+const BounceableStyle = styled(Bounceable)<{
+  outer: {width: number; height: number; border: number}
+  ringColor: string
+  disable: boolean
+  disableOpacity: number
+}>(({outer, ringColor, disable, disableOpacity}) => ({
+  width: outer.width,
+  height: outer.height,
+  borderWidth: metrics.borderRadius,
+  borderRadius: outer.border,
+  borderColor: ringColor,
+  alignItems: 'center',
+  justifyContent: 'center',
+  opacity: disable ? disableOpacity : 1,
+  backgroundColor: 'transparent',
+}))
+
+const RadioButtonWrapper = styled(View)({
+  flexDirection: 'row',
+  alignItems: 'center',
+})
+
+const RadioButtonInnerContainer = styled(View)<{
+  inner: {width: number; height: number; border: number}
+  maxWidth: number
+  maxHeight: number
+  isActive: boolean
+  innerBackgroundColor: string
+}>(({inner, maxWidth, maxHeight, isActive, innerBackgroundColor}) => ({
+  maxWidth: maxWidth,
+  maxHeight: maxHeight,
+  width: inner.width,
+  height: inner.height,
+  borderRadius: inner.border,
+  backgroundColor: isActive ? innerBackgroundColor : 'transparent',
+}))
+
+const LabelTextView = styled(View)<{disable: boolean; disableOpacity: number}>(
+  ({disable, disableOpacity}) => ({
+    marginLeft: responsiveWidth(16),
+    opacity: disable ? disableOpacity : 1,
```

Comment:
use OPACITY_DEFAUTL defined above instead of 1

Reply:
already set disableOpacity = OPACITY_DEFAULT

Reply:
OPACITY_DEFAULT setting 0.5

Reply:
I think `opacity = 1` is a basic value, everyone can understand.